modules/auto-delete/voice-channels.json

```diff
@@ -0,0 +1,27 @@
+{
+  "filename": "voice-channels.json",
+  "humanname-de": "Voice-Channel",
+  "humanname-en": "Voice-Channels",
+  "configElements": true,
+  "content": [
```

Comment:
I highly suggest adding the humanName attributes to each config field, to improve user experience (see https://github.com/SCNetwork/CustomDCBot/blob/ebefb00b3aadda8dbb693efa507ebe1ffc8e7d07/modules/betterstatus/config.json#L8 for reference)

Reply:
added detailed descriptions for both auto-delete functionalities.